### PR TITLE
Update select Secrets of Magic Battle Form spells per Paizo Fall 2024 errata

### PR DESCRIPTION
**Pull request by ammalagonc**

### CODE REVIEW

File: packs/spell-effects/spell-effect-devil-form-levaloch.json [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+                "key": "FlatModifier",
+                "label": "PF2E.BattleForm.DevilForm.LevalochSavesAgainstMoveLabel",
+                "predicate": [
+                    "resist-move"
+                ],
+                "selector": "saving-throw",
+                "type": "circumstance",
+                "value": 2
+            },
+            {
```
Comment (TikaelSol) [inline packs/spell-effects/spell-effect-devil-form-levaloch.json:122]:
```suggestion
            },
            {
                "key": "FlatModifier",
                "label": "PF2E.BattleForm.DevilForm.LevalochSavesAgainstMoveAllLabel"
                "predicate": [
                    {
                        "or": [
                            "action:shove",
                            "resist-move"
                        ]
                    }
                ],
                "selector": "saving-throw",
                "type": "circumstance",
                "value": 2
            }
            {
```
We can condense it

File: packs/spell-effects/spell-effect-devil-form-levaloch.json [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+                "category": "precision",
+                "diceNumber": 2,
+                "dieSize": "d6",
+                "hideIfDisabled": true,
+                "key": "DamageDice",
+                "predicate": [
+                    "item:range-increment:20",
```
Comment (TikaelSol) [inline packs/spell-effects/spell-effect-devil-form-levaloch.json:129]: Why the range increment? Shouldn't it be for all melee strikes? Which would be just the melee strikes from the trident.

File: static/lang/re-en.json [hunk 3]
```diff
@@ -171,7 +173,13 @@
                 "SunNote": "The target is @UUID[Compendium.pf2e.conditionitems.Item.TkIyaNPgTZFBCCuh]{Dazzled} for 1 round."
             },
             "DaemonForm": {
-                "CeustodaemonDamageNote": "Any successful jaws or claw Strike deals an additional @Damage[1d6[untyped]] damage, and you take the same amount of damage"
+                "AbrikandiluJawsCriticalHitNote": "On a critical hit, the target is @UUID[Compendium.pf2e.conditionitems.Item.i3OJZU2nk64Df3xm]{Clumsy 1} for 1 minute.",
+                "CeustodaemonDamageNote": "Any successful jaws or claw Strike deals an additional @Damage[1d6[untyped]] damage, and you take the same amount of damage",
+                "KithangianPincerGrabNote": "You can spend an action after a hit to attempt to @UUID[Compendium.pf2e.bestiary-ability-glossary-srd.Item.Tkd8sH4pwFIPzqTr]{Grab} the target."
+            },
+            "DevilForm": {
+                "LevalochFortitudeAgainstShoveLabel": "Against being Shoved",
```
Comment (TikaelSol) [inline static/lang/re-en.json:181]:
```suggestion
```
Shouldn't need both, and I think the other could maybe go to some place more generic.

File: packs/spell-effects/spell-effect-devil-form-levaloch.json (hunk 2 above)
Comment (ammalagonc) [reply packs/spell-effects/spell-effect-devil-form-levaloch.json:121]: I read it wrong - this was my last entry, and my eyes were already screaming by that point.
Fixing

Comment (TikaelSol) [reply packs/spell-effects/spell-effect-devil-form-levaloch.json]: I hear that

File: static/lang/re-en.json [hunk 4]
```diff
@@ -171,7 +173,12 @@
                 "SunNote": "The target is @UUID[Compendium.pf2e.conditionitems.Item.TkIyaNPgTZFBCCuh]{Dazzled} for 1 round."
             },
             "DaemonForm": {
-                "CeustodaemonDamageNote": "Any successful jaws or claw Strike deals an additional @Damage[1d6[untyped]] damage, and you take the same amount of damage"
+                "AbrikandiluJawsCriticalHitNote": "On a critical hit, the target is @UUID[Compendium.pf2e.conditionitems.Item.i3OJZU2nk64Df3xm]{Clumsy 1} for 1 minute.",
+                "CeustodaemonDamageNote": "Any successful jaws or claw Strike deals an additional @Damage[1d6[untyped]] damage, and you take the same amount of damage",
+                "KithangianPincerGrabNote": "You can spend an action after a hit to attempt to @UUID[Compendium.pf2e.bestiary-ability-glossary-srd.Item.Tkd8sH4pwFIPzqTr]{Grab} the target."
+            },
```
Comment (TikaelSol) [inline static/lang/re-en.json:179]: To cut down on the long properties here let's make this `DaemonForm.Notes.AbrikandiluCritical` etc.

File: static/lang/re-en.json [hunk 5]
```diff
@@ -171,7 +173,12 @@
...
+            "DevilForm": {
+                "LevalochSavesAgainstMoveAllLabel": "Resist being moved against your will"
```
Comment (TikaelSol) [inline static/lang/re-en.json:181]: Maybe move this to `PF2E.GenericLabel.Resist.ForcedMovement` and just make it `vs. forced movement`

File: static/lang/re-en.json (hunk 4 above)
Comment (TikaelSol) [reply static/lang/re-en.json]: Since it's an effect  no migration needed

**Pull request merged**